tests/gpu/GPUUnitTest.cpp

```diff
@@ -5,7 +5,6 @@
 #include <algorithm>
 #include <cmath>
 #include <cstring>
-#include <iomanip>
```

Comment:
Minor: why do we remove iomanip here? Seems like this file uses std::setprecision.

Reply:
That was my editor being overzealous with auto format again. I reverted the whitespace changes but missed this one. Reverted this as well.